### ISSUE TITLE
Verify Accessibility Compliance TestCase

### ISSUE THREAD

**Issue by sayom88** [sayom88/GenAIDocs#646]
TC No: TC_009\nTestCase Name: Verify Accessibility Compliance\nPre-Requisites: The application should be WCAG 2.1 compliant.\
